src/gdbstub_impl/ext/base.rs

```diff
@@ -615,7 +571,7 @@ impl<T: Target, C: Connection> GdbStubImpl<T, C> {
     }
 
     #[inline(always)]
-    fn finish_exec(
+    pub(crate) fn finish_exec(
```

Comment:
Any reason why this needs to be `pub(crate)`?

Reply:
I'm reusing this code in `reverse_exec.rs` to send back a thread stop reason.

Reply:
Oh, maybe I have my crate visibility rules mixed up, but I thought this would have worked without it...
If not, all good.

That said, can you check if `pub(super)` works? If it does, it'd be a bit more appropriate.

Reply:
Indeed that works! I'll swap it over.